README.md

```diff
@@ -2,23 +2,23 @@ Welcome to X Layer
 
 X Layer is a ZK L2 network powered by the OKB token and built using Polygon CDK, that offers 100% EVM compatibility, exceptional performance, low fees, and robust security — achieved with seamless integration.
 
-## zkEVM RPC Versions
-The following table shows the recommended versions to use for permitssionless nodes:
+## Erigon RPC Version [Recommend]
 
-| Network   | Node      | Prover/Executor   |
-| ---       | ---       | ---               |
-| Mainnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |
-| Testnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |
+| Network   |             Erigon       |
+| ---       | ---                      |
+| Mainnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
+| Testnet   | [v2.61.0](https://github.com/0xPolygonHermez/cdk-erigon/releases/tag/v2.61.0)    |
 
+## zkEVM RPC Version [Will be deprecated]
 
-## Erigon RPC Versions
-Please refer to [cdk-erigon](https://github.com/0xPolygonHermez/cdk-erigon)
+| Network   | Node      | Prover/Executor   | 
+| ---       | ---       | ---               | 
+| Mainnet   | [v0.4.0](https://github.com/okx/xlayer-node/releases/tag/v0.4.0)    |  [v0.3.2](https://github.com/okx/xlayer-prover/releases/tag/v0.3.2)    |   
```

Comment:
Our xlayer-node also has [v0.4.1](https://github.com/okx/xlayer-node/releases/tag/v0.4.1), are we planning to deploy that?